delta/description.adoc

```diff
@@ -73,20 +73,51 @@ The repository also sends out the same events to subscribed clients, thus _not_
 
 [[message, message]]
 === Message characteristics
-Each message is atomic; it cannot be split.
 
+[[messageAtomicity]]
+.Atomicity
+Most message is atomic; they cannot be split.
+The exceptions are all messages containing a <<chunkType>>.
+
+[[chunkedMessages]]
+.Chunked messages
+
+Queries::
+* <<qry-SubscribeToPartitionContents>> <<SubscribeToPartitionContents.contents>> response
+* <<qry-ListPartitions>> <<ListPartitions.partitions>> response
+
+Commands::
+* <<cmd-AddPartition>> <<AddPartition.newPartition>>
+* <<cmd-AddChild>> <<AddChild.newChild>>
+* <<cmd-ReplaceChild>> <<ReplaceChild.newChild>>
+* <<cmd-AddAnnotation>> <<AddAnnotation.parent>>
+* <<cmd-ReplaceAnnotation>> <<ReplaceAnnotation.newAnnotation>>
+
+Events::
+* <<evnt-PartitionAdded>> <<PartitionAdded.newPartition>>
```

Comment:
Because of [https://github.com/LionWeb-io/specification/issues/425](https://github.com/LionWeb-io/specification/issues/425) this PartitionAdded chunk will always be one node, so it never becomes too big.